util/etcd/etcd.go

```diff
@@ -0,0 +1,368 @@
+/*
```

Comment:
Is this the best place to put this package? And do we want to expose it for others to use it as well or make it internal to the kubeadm control plane for the time being?

Reply:
Moved